cmd/hyperkube/hyperkube.go

```diff
@@ -22,7 +22,7 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
-	"os"
+				"os"
```

Comment:
Seems like a typo?

Reply:
It is. This is part of my commit that breaks things. :)